file.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package go_sftp
+
+import (
+	"io"
+	"io/fs"
+
+	"github.com/pkg/sftp"
+)
+
+type File struct {
+	*sftp.File
+
+	Filename string
+	Contents io.ReadCloser
+}
+
+func (f File) Close() error {
```

Comment:
Isn't there also a `Close()` on the `sftp.File`? We should close it here as well if so.

Reply:
Yea. I'm going to keep the behaviour of `Open(..)` reading the file contents, but we'll pass through the `fs.File` methods.